### ISSUE TITLE
Module positions...

### ISSUE THREAD

**Issue by Dorvador** [GavickPro/Music-Free-Responsive-Joomla-Template#193]
Hello! 

I installed your very good template "Music Pro" which in my humble opinion, has only one flaw: I can not find all module positions as shown on this page : http://demo.gavick.com/joomla25/music/index.php/template/lorem-ipsum/module-positions
Can I have all these module positions available! 
Thank you for your help. 

Ilan

**Comment by bkrztuk** [comment]
I think that you mix up templates. The image comes from Music template (commercial) and you wrote on Music Free repository. If you use **Music Free** the available module positions you can check here →  [Module Positions](http://demo.gavick.com/free/may2012/index.php/template/lorem-ipsum/template-articles/module-positions)

**Comment by Dorvador** [comment]
thank you for your reply,

this means that if I buy a paid version of "music pro" I will have all the
module positions?
And if this is the case, will it be easy to upgrade my site or do I have to
redo everything?

Thank you!

2014-06-30 14:50 GMT+03:00 Bartłomiej Krztuk notifications@github.com:

**Comment by bkrztuk** [comment]
No, there is no PRO version of the template. This are totally different products one is commercial and the second one is free but this is different template. (there is no way to upgrade to different product)
